### ISSUE TITLE
dgraph  20.03.0 live panic: interface conversion: interface {} is nil, not int64

### ISSUE THREAD

**Issue by zhaolin-st** [dgraph-io/dgraph#5386]
I was using live in Dgraph version 20.03.0, problem was same like  Issue #4889
https://github.com/dgraph-io/dgraph/issues/4889
There is a "age [Int] "in the schema.

![image](https://user-images.githubusercontent.com/30283563/81274382-cc8a8600-9082-11ea-86df-f5d51d2cd3b6.png)

**Comment by danielmai** [comment]
Can you share an example of the JSON and schema used for live loader?

**Comment by zhaolin-st** [comment]
Data and schema like this
```
<age>: [int] @index(term) .
<name>: [string] @index(hash) .
```

```
{
    "set": [
        {
            "name": [
                "诸暨"
            ],
            "uid":  "_:abc"
            "age": [
                23
            ]
        }
      ]
}


```

> Can you share an example of the JSON and schema used for live loader?

**Comment by PabloCastellano** [comment]
I've hit the same issue with v20.03.0 but it looks like the bug is fixed in v20.03.3

**Comment by minhaj-shakeel** [comment]
**Github issues have been deprecated.** 
This issue has been moved to [discuss](https://discuss.dgraph.io/t/8524). You can follow the conversation there and also subscribe to updates by changing your notification preferences.

 <img src="https://github.com/dgraph-io/benchmarks/blob/master/images/SetNotifications.png?raw=true" alt="drawing" width="40%"/>
